### DISCUSSION TITLE
ERC 721 Strategies

### DISCUSSION THREAD

**Discussion by hirokennelly** [snapshot-labs/snapshot#1199] [Q&A] [answered]
I see you can quadratically weight votes, but can you also quadratically weight the contents of a wallet? For ex, under an ERC 721 vote where some people hold multiple NFTs, can we quadratically weight the number of NFTs for voting purposes?

Can we limit the vote to one vote per wallet, even if that wallet has multiples NFTs?

**Comment by ChaituVR** [comment, marked as answer]
- Yes you can use multiple NFTs and quadratically weight them with erc721 strategy
- For "one vote for any number of NFTs" you can use `erc20-with-balance` strategy
https://snapshot.org/#/strategy/erc20-with-balance
